modules/seqexec/server/src/main/scala/seqexec/server/SeqexecEngine.scala

```diff
@@ -193,18 +193,27 @@ class SeqexecEngine(httpClient: Client[IO], settings: Settings[IO], sm: SeqexecM
 
   def requestRefresh(q: EventQueue, clientId: ClientId): IO[Unit] = q.enqueue1(Event.poll(clientId))
 
-  def seqQueueRefreshStream: Stream[IO, executeEngine.EventType] =
-    Scheduler[IO](corePoolSize = 1).flatMap { scheduler =>
-      val fd = Duration(settings.odbQueuePollingInterval.toSeconds, TimeUnit.SECONDS)
-      scheduler.fixedDelay[IO](fd).evalMap(_ =>
-        odbProxy.queuedSequences.value).map { x =>
-        Event.getState[executeEngine.ConcreteTypes](st =>
-          x.map(odbLoader.refreshSequenceList(_)(st)).valueOr(r =>
-            List(Event.logWarningMsg(SeqexecFailure.explain(r)))
-          ).some.filter(_.nonEmpty).map(Stream.emits(_).covary[IO])
-        )
-      }
+  def seqQueueRefreshStream: Stream[IO, executeEngine.EventType] = {
+    val fd = Duration(settings.odbQueuePollingInterval.toSeconds, TimeUnit.SECONDS)
+    Stream.fixedDelay[IO](fd).evalMap(_ => odbProxy.queuedSequences.value).map { x =>
+      Event.getState[executeEngine.ConcreteTypes](st =>
+        x.map(odbLoader.refreshSequenceList(_)(st)).valueOr(r =>
+          List(Event.logWarningMsg(SeqexecFailure.explain(r)))
+        ).some.filter(_.nonEmpty).map(Stream.emits(_).covary[IO])
+      )
+  // def seqQueueRefreshStream: Stream[IO, executeEngine.EventType] =
+  //   Scheduler[IO](corePoolSize = 1).flatMap { scheduler =>
```

Comment:
Do you know why we don't need the `Scheduler` anymore?

Reply:
It was folded into cats-effect `Timer`